src/deepsparse/transformers/haystack/nodes.py

```diff
@@ -298,7 +295,15 @@ def embed_documents(self, docs: List[Document]) -> List[numpy.ndarray]:
         :param docs: list of document strings to embed
         :return: list of embeddings for each document
         """
-        passage_inputs = [doc.content for doc in docs]
+        passage_inputs = [
+            f"{doc.meta['title']}{self.title_separator}{doc.content}"
```

Comment:
Is prepending title before content with a character between pretty standard for including title?

Reply:
Yes I've seen it done this way by Haystack as well